### PR TITLE
added comments for SMS examples and changed send() to end() in handleInboundSms()

### PR DESCRIPTION
**Pull request by ukitta555**
I thought that maybe new developers might need some help in understanding example code. I did come commenting about express and JS so that code makes more sense for people that aren't so confident in their JS skills.

### CODE REVIEW

File: sms/receive-express.js [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
 function handleInboundSms(request, response) {
+  /*
+  *  Merge request.query with request.body object
+  *  and assign the result to params variable
+  */
   const params = Object.assign(request.query, request.body)
   console.log(params)
+  // since nothing is returned, send 204 code and use end() method
+  // https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
   response.status(204).send()
```
Comment (garann) [inline sms/receive-express.js:28]: This references `end()` in the comment but is using `send()`. Maybe this one got missed?

**Comment by kellyjandrews** [comment]
Thanks for these @ukitta555 - I'll have a look later this week.

**Comment by juliacodez** [comment]
@ukitta555 Thank you for your contribution! 

While these detailed code comments could help with understanding our code snippets, I'm afraid they would also detract from their main usage on the developer portal. 

@kellyjandrews I suggest applying the hacktoberfest-accepted tag here, and closing the PR without merging.

**Comment by juliacodez** [comment]
Thank you for taking the time to contribute to our code snippets! 

While these detailed code comments could help with understanding our code snippets, I'm afraid they would also detract from their main usage on the developer portal. Unfortunately we won't be able to merge, as this would conflict with the standards we currently have across all code snippets.

**Pull request closed without merging**